classes/class-wc-connect-shipping-label.php

```diff
@@ -73,11 +73,32 @@ protected function get_packaging_data( WC_Order $order ) {
 			return $packages;
 		}
 
+		protected function get_selected_rates( WC_Order $order ) {
+			$shipping_method = reset( $order->get_shipping_methods() );
```

Comment:
Sure, I'll store it in a temp variable. Which PHP version are you on? I'm in `5.6.21` and I had no warning even with `define('WP_DEBUG', true);`.